model/SearchEngine/Driver/Elasticsearch/ElasticSearchConfig.php

```diff
@@ -26,7 +26,7 @@
 
 use oat\generis\model\DependencyInjection\ServiceOptionsInterface;
 
-class ElasticSearchConfig
+class ElasticSearchConfig implements ElasticSearchConfigInterface
```

Comment:
Why do we need this?

Reply:
As we have now two implementations for getting configs